.github/workflows/build.yml

```diff
@@ -189,3 +189,7 @@ jobs:
           MC_TWITTER_ACCESS_TOKEN_SECRET: ${{ secrets.MC_TWITTER_ACCESS_TOKEN_SECRET }}
           MC_TWITTER_CONSUMER_KEY: ${{ secrets.MC_TWITTER_CONSUMER_KEY }}
           MC_TWITTER_CONSUMER_SECRET: ${{ secrets.MC_TWITTER_CONSUMER_SECRET }}
+
+      - name: List test failures and errors
+        run: echo -e 'Test failures:\n' && awk '/FAILURES/,/short/' /home/runner/runners/*/_diag/pages/*.log && echo -e '\n\nErrors:\n' && awk '/non-zero/,/status 1/' /home/runner/runners/*/_diag/pages/*.log
```

Comment:
What's in `/home/runner/runners/*/_diag/pages/*.log`, i.e. how are you finding those errors? Do you perhaps have a sample log file from that directory that we could look at (without various private credentials that might be in this log)?

Grepping (awking?) out failures from a log file is a bit wonky because any test might print a string `FAILURES` at any point for a legitimate reason and still succeed; plus, tests don't have to fail with `status 1` specifically as they can have any non-zero exit status.